agent/agent-tooling/build.gradle.kts

```diff
@@ -25,6 +25,17 @@ dependencies {
     exclude("org.ow2.asm", "asm")
   }
 
+  implementation("ch.qos.logback:logback-classic")
+  implementation("ch.qos.logback.contrib:logback-json-classic")
+
+  // not using gson because it has dependency on java.sql.*, which is not available in Java 9+ bootstrap class loader
+  // only complaint so far about moshi is that it doesn't give line numbers when there are json formatting errors
+  implementation("com.squareup.moshi:moshi")
```

Comment:
this was what motivated moving this code out of the bootstrap module (in addition to it being a generally good thing to minimize amount of classes in the bootstrap module)